Source/CarthageKit/Project.swift

```diff
@@ -8,9 +8,6 @@ import XCDBLD
 import ReactiveTask
 import SPMUtility
 
-import struct Foundation.URL
```

Comment:
If per-type imports are preferred, we should add more Foundation types here (I don't check what types are required in this file).